building-epics/cross-compile-epics-and-a-ioc-to-an-old-x86-linux.md

```diff
@@ -0,0 +1,378 @@
+# How To cross compile EPICS and a IOC to an old x86 Linux system
+
+## Introduction
+
+I was given the task of developing a IOC which should run in a x86 PC with an old Linux distribution.
+My development machine was a x86_64 PC running Ubuntu 12.04.
+EPICS does a great job cross compiling from a 64-bits host to a 32-bits target if both have compatible versions of glibc,
+binutils,
+kernel,
+etc.
+In my case,
+however,
+my target had much older versions. I considered two different solutions:
```

Comment:
I would write it like that, but of course there's a bit of interpretation:

```suggestion
I was given the task of developing a IOC
which should run in a x86 PC with an old Linux distribution.
My development machine was a x86_64 PC running Ubuntu 12.04.
EPICS does a great job cross compiling from a 64-bits host to a 32-bits target
if both have compatible versions of glibc, binutils, kernel, etc.
In my case, however,
my target had much older versions.
I considered two different solutions:
```

I *think* those `which` and `if` introduce a relative clause, so it's good to have a line break before. If I'm note sure, I just usually cheat and add a comma.